torch/csrc/fx/fx_init.cpp

```diff
@@ -137,6 +159,138 @@ void initFx(PyObject* module) {
   if (PyModule_AddObject(module, "_fx", patch) != 0) {
     throw python_error();
   }
+
+  auto m = py::handle(module).cast<py::module>();
+  auto key = m.def_submodule("key");
+  key.def("addKey", &addKey, py::return_value_policy::copy); // not sure if needed - cargo cult
+  key.def("removeKey", &removeKey);
+  key.def("hasKey", &hasKey);
+}
+
+
+
+py::object pyIdentity(py::object x) {
+  return x;
+}
+template<class T>
+py::tuple vectorToPyTuple(const std::vector<T> &data, std::function<py::object(T)> converter) {
+	PyObject* tuple = PyTuple_New( data.size() );
+	if (!tuple) throw std::runtime_error("Unable to allocate memory for Python tuple");
+	for (unsigned int i = 0; i < data.size(); i++) {
+		PyObject *num = converter(data[i]).ptr();
+		if (!num) {
+			Py_DECREF(tuple);
+			throw std::runtime_error("Unable to allocate memory for Python tuple");
+		}
+    Py_INCREF(num); // todo: dunno?? Need it to fix segfaults, but probably not right
+		PyTuple_SET_ITEM(tuple, i, num);
+	}
```

Comment:
don't use tabs

Reply:
Oh, this used to have a [WIP][not for review] on it, before I switched to ghstack.